### DISCUSSION TITLE
Replace AI generated loading and title screen with cooler stuff that fit better

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#14276] [Ideas and suggestions]
Weird, AI generated images aren't really good to have as the first thing you see when you load into the game.

They aren't interesting to look at, mostly just confusing (because they are incoherent)
It looks like *not barotrauma*, and they are just mediocre. Even a badly done photobash is much more interesting, and the game has plenty of cool art. Maybe the game needs new art, to prevent just reusing the same art everywhere. 

Having a bunch of "generic AI generated submarine" really does not sell the vibe of barotrauma. Please add more art.

A link to the discord forum on the topic, which many people have shared their opinions in:
https://canary.discord.com/channels/103209684680323072/1260663354620645446/1260663354620645446

**Comment by girlyguppy** [comment]
here's a thread of cool ass art, followed by weird blobby "edits"
![Barotrauma_Wallpaper_02](https://github.com/FakeFishGames/Barotrauma/assets/87497640/6cf29a26-528b-4e8c-80fa-843e1ad78c1a)
![HabitationOutpost3](https://github.com/FakeFishGames/Barotrauma/assets/87497640/700b9b8b-c2ef-4f82-bfe0-5f70ad0b2167)

**Reply by girlyguppy** [reply]
first images have barotrauma stuff (poverty, baro outfits, assassination posters, diving suits etc)
second image has a generic scifi building with nothing that seems at all barotrauma, aside from it being a building (barotrauma has buildings!!!)
![Barotrauma_Wallpaper_01](https://github.com/FakeFishGames/Barotrauma/assets/87497640/a34f8323-d142-4ef9-9310-d04ebeede6fa)
![WorkCampMilitary4](https://github.com/FakeFishGames/Barotrauma/assets/87497640/0b108c4f-7a32-43b2-a85f-347811967f3d)

**Reply by girlyguppy** [reply]
![Barotrauma_Wallpaper_09](https://github.com/FakeFishGames/Barotrauma/assets/87497640/eda3e89f-923f-4cca-9304-5ee415606ab8)
![Colony3](https://github.com/FakeFishGames/Barotrauma/assets/87497640/edd17784-d99f-4a01-b5ba-2c9d6a93f872)

**Comment by itchyOwl** [comment]
Thanks for the feedback, I'll forward it. Afaik all of those images are authored by our artists. I don't know what exactly the work process has been, but I know the images are certainly not AI generated. In-game, there's some extra layers with some procedural stuff, but the background images can be found under `Content/Map/LocationPortraits`. And the reference images are found in the supporter pack, under `bonus content/Wallpapers`.

If I understand it correctly, the loading images are working properly but you just liked the versions found in the supporter pack more? I think they might have been some earlier versions of those images we decided to use. I can't really say I agree with you that they are better than those we use. But I do agree that we could maybe add some variety!

**Reply by girlyguppy** [reply]
I am not saying that the images are entirely created by AI, just that they have been fed through an AI in an effort to (???), which had the effect of removing game-relevant details that were in the original images. It removes immersion that the original images would provide us. 
![image](https://github.com/user-attachments/assets/90594dcc-25a5-4175-b86b-8f3d802aeca0)
^ signed, the discord server

they go from highly detailed barotrauma photobashes to "weird nondescript painting", regardless of the tool used to do it.
but the lack of coherence makes it unmistakably AI-related. it's covered in artifacts, and no other tool that I know of makes pictures that look like this. As an artist, I can't think of any artist who lost the ability to draw straight lines and shapes that could exist in our dimension.
![image](https://github.com/user-attachments/assets/62f1cf45-d294-49e9-b4d8-0891639aa8df)
ah yes, the biomechanical amalgamation. from barotrauma.


workcampmining 3 and workcampresearchfacilicty3 is great, it's clear what it is. everything in the image makes sense, and is clearly barotrauma. endportrait1 is also clearly humanmade.
![WorkCampResearchFacility3](https://github.com/user-attachments/assets/6ac1809a-203b-4c84-99d4-6c576c39bbfa)
![WorkCampMining3](https://github.com/user-attachments/assets/358fb4d1-bfd8-4e0f-999b-26df79a4cf25)
 we should use more of these non-AI ones. these are the only ones i can find that do make sense unfortunately.

**Reply by girlyguppy** [reply]
![image](https://github.com/user-attachments/assets/076da1a1-3fe6-4a4b-b621-4cf4453abf09)

do you prefer messed up gauge
OR

![image](https://github.com/user-attachments/assets/21b1cc29-a475-40d3-a1d0-915136d9f684)

an actual junction box  from the game barotrauma.

**Reply by girlyguppy** [reply]
![image](https://github.com/user-attachments/assets/6ec5eb9f-c57c-484c-abd2-743d6e8eb5f3)
the original image at least the ladder isn't phasing into the 5th dimension

**Reply by girlyguppy** [reply]
![image](https://github.com/user-attachments/assets/afd9b528-0504-4bf9-953e-2407c5de54f4)
left-right flipping this AI location image shows better the 1:1 with this bonus content wallpaper
![Barotrauma_Wallpaper_09](https://github.com/user-attachments/assets/827a3130-a91f-4465-b172-c7a29eea7bbf)

**Reply by girlyguppy** [reply]
ultra-detailed artwork showing the outposts, poverty, grunginess, it even has bounty posters. amazing.

too bad it gets turned into generic space building with a total of 1 coherent details: this person
![image](https://github.com/user-attachments/assets/8d1f9a3f-8efa-4282-8406-08c1b760360e)

**Reply by itchyOwl** [reply]
Thanks for the feedback. Seems that you are right in that AI generation has been used and it's been on the todo list of our artists to clean up the loading art. I'm a programmer and not an artist, so I don't have a trained eye for these things ;)

**Comment by Max112358** [comment]
Well I for one think the Barotrauma art is half decent. Its good enough for a loading screen. It does look AI generated, but I'm ok with that. Although if your going to use AI generated art (or something so close to that I cant tell the difference) you might as well make hundreds of them.

**Reply by girlyguppy** [reply]
Ah yes, "half decent" and "good enough", the epitome of AI

But seriously, the wallpapers are 10x better, they should use them. They add way more.

**Comment by Nathan-Julian** [comment]
I personally love seeing awesome art created with real intent. This is especially true for barotrauma, which has such a cool and unique universe and holds so much potential for art to improve it further. Don't squander the amazing world you've created by representing it with algorithmically generated nonsense!

Additionally, I just hate consuming ai-generated "art". It leaves such a terrible taste in my mouth to know that so little thought or effort was put into the image, not to mention the massive issues with web scraping and such associated with image generation.

**Comment by Arctodites** [comment]
I quite like the concept art for this game, especially the exterior scenes.  It gives a feel for the creators' vision of life on Europa.  IIRC, a lot of these images were created before generative AI was good enough to imitate this style of art.

Even if the concept pieces are a bit sterile, they evoke more possibilities. I only wish that the in-game outposts had more life to them: NPCs perforning routine tasks with prop items (e.g. welding, carrying boxes, smoking, drinking a beer, reading a book, preparing food at a cafeteria stall ...).  It's also uncanny that there are no children or animals running around.  Even if there were just a greater variety of harmless sealife growing around the outposts, that would help make the world feel more expansive.

**Comment by Regalis11** [comment]
These AI-generated loading screens have been replaced now in the latest update, closing. :)
